protocol/kafka_sarama/v2/go.mod

```diff
@@ -7,7 +7,7 @@ replace github.com/cloudevents/sdk-go/v2 => ../../../v2
 require (
 	github.com/Shopify/sarama v1.19.0
 	github.com/Shopify/toxiproxy v2.1.4+incompatible // indirect
-	github.com/cloudevents/sdk-go/v2 v2.0.0-00010101000000-000000000000
```

Comment:
The release automation needs to update the modules to point to the new release before releasing them.